tests/test_main.py

```diff
@@ -339,3 +339,19 @@ def test_use_coverage(capsys, filesystem):
         assert '7/7  🎉 7  ⏰ 0  🤔 0  🙁 0' in repr(result.output)
     else:  # python2
         assert '8/8  \\U0001f389 8  \\u23f0 0  \\U0001f914 0  \\U0001f641 0' in repr(result.output)
+
+
+def test_pre_and_post_mutation_hook(filesystem):
+    result = CliRunner().invoke(
+        climain, [
+            'run',
+            '--paths-to-mutate=foo.py',
+            "--test-time-base=15.0",
+            "--pre-mutation=echo pre mutation stub",
+            "--post-mutation=echo post mutation stub",
+        ], catch_exceptions=False)
+    print(result.output)
```

Comment:
Looks like this `print` is not required

Reply:
It isn't, but when the test fails it's super nice to have.